DashKitUI/Views/NotificationSettingsView.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  NotificationSettingsView.swift
+//  DashKitUI
+//
+//  Created by Pete Schwamb on 2/3/21.
+//  Copyright © 2021 Tidepool. All rights reserved.
+//
+
+import SwiftUI
+import LoopKit
+import LoopKitUI
+import HealthKit
+
+struct NotificationSettingsView: View {
+    
+    var dateFormatter: DateFormatter
+    
+    @Binding var expirationReminderDefault: Int
+    
+    @State private var showingHourPicker: Bool = false
+    
+    var scheduledReminderDate: Date?
+    
+    var allowedReminderDateRange: ClosedRange<Date>
+    
+    var lowReservoirReminderValue: Int
+    
+    var onSaveScheduledExpirationReminder: ((_ selectedDate: Date, _ completion: @escaping (_ error: Error?) -> Void) -> Void)?
+    
+    var onSaveLowReservoirReminder: ((_ selectedValue: Int, _ completion: @escaping (_ error: Error?) -> Void) -> Void)?
+    
+    var insulinQuantityFormatter = QuantityFormatter(for: .internationalUnit())
+
+    var body: some View {
+        RoundedCardScrollView {
+            RoundedCard(
+                title: "Omnipod Reminders",
+                footer: "The App notifies you in advance of Pod expiration.  Set the number of hours advance notice you would like to have."
+            ) {
+                expirationReminderRow
+                if let scheduledReminderDate = scheduledReminderDate  {
+                    Divider()
+                    scheduledReminderRow(scheduledDate: scheduledReminderDate)
+                }
+            }
+
+            RoundedCard(footer: "The App notifies you when the amount of insulin in the Pod reaches this level.") {
```

Comment:
[nit] "The App"?

Reply:
That's how it is in the design. I think it's fine; but would be easy to change to app name if we want.

Reply:
Nope, not necessary.